Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as

```diff
@@ -142,6 +142,15 @@ package com.flashlight.vnc
 			}
 		}
 
+		private static function jsLog(message:String):void {
+		  if (ExternalInterface.available) {
+		    try {
+		      ExternalInterface.call("flashLog", message);
+		    } catch (e:Error) {
```

Comment:
As a [wise man once said to me](https://github.com/rainforestapp/Flashlight-vnc/pull/21#issuecomment-197282409):
> silently catching errors can get tricky down the road

LOL